### ISSUE TITLE
Ensure implicit list markup is avoided

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#67]
## Description
Whenever lists are utilized in a page, they should incorporate the appropriate markup for lists. Lists which are created using tables, indentation, or with images as bullet points should be avoided. Instead, developers should utilize appropriate list markup and cascading style sheets to format lists.

### CTA Grid, Related Content, News Grid, Jobs Board, Staff Profiles
- [x]  Grids should be `ul`s so that screen reader users won't lose the semantic information the structure of the list provides visual users.

### Pagination
- [x] Page number links in pagination nav should be `ul`.

<img width="918" alt="fe0533b500c6055ce5b424e755cf1306d44650b3-1641282456" src="https://user-images.githubusercontent.com/8166060/153069516-6be9473f-3770-4bcb-81d9-2509177eb5ef.png">

Compliant Code Example:
```
<ul>
  <li>1  </li>
  <li>
    <a tabindex="0" href="/gallery?page=2">2</a>
  </li>
  <li>
    <a tabindex="0" href="/gallery?page=3">3</a>
  </li>
  <li>
    <a tabindex="0" href="/gallery?page=4">4</a>
  </li>
</ul>
```

## CIC Estimate
1 point
